frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -241,17 +240,16 @@ export const insightLogic = kea<insightLogicType>({
                     // fetch this now, as it might be different when we report below
                     const scene = sceneLogic.isMounted() ? sceneLogic.values.scene : null
 
-                    // If a query is in progress, debounce before making the second query
+                    // If a query is in progress, kill that query
                     if (cache.abortController) {
-                        await breakpoint(300)
```

Comment:
Without breakpoint:
<img width="668" alt="Pasted Graphic" src="https://user-images.githubusercontent.com/1727427/186530223-65709d79-7d5f-437c-8140-471397d9cf80.png">

With breakpoint (set to 2 seconds to exaggerate)
<img width="423" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/1727427/186530236-76d53a04-6b20-4fcf-8dd7-b4d5f5228ebc.png">

Because we're now cancelling queries, we can fire off these queries quicker, which will mean quicker results. Open to debate on this.